src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -576,6 +576,15 @@ OperatorsBridge::OperatorsBridge() {
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "SkipLayerNormalization", 1, skip_layer_normalization);
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "Trilu", 1, trilu);
 
+    register_operator_in_custom_domain("DequantizeLinear",
+                                       VersionRange::since(1),
+                                       op::set_13::dequantize_linear,
+                                       "com.microsoft");
+    register_operator_in_custom_domain("QuantizeLinear",
+                                       VersionRange::since(1),
+                                       op::set_13::quantize_linear,
+                                       "com.microsoft");
+
```

Comment:
`MICROSOFT_DOMAIN` contains `com.microsoft`. Can we use `REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "DequantizeLinear", 1, op::set_13::dequantize_linear);` as it was utilized for other Microsoft operators?

Reply:
Nop, original macro will produce "op::set_1::op::set_13::quantize_linear"